requirements/runtime.txt

```diff
@@ -123,8 +118,6 @@ rich==13.9.4
     # via
     #   sigstore
     #   twine
-secretstorage==3.3.3
-    # via keyring
```

Comment:
NB: Not sure why this dropped, since I didn't touch it. But I _think_ it shouldn't matter since this action doesn't touch the keyring bits of `twine` at all?

Reply:
Fixed!